celer/cython_utils.pyx

```diff
@@ -187,26 +192,34 @@ cdef floating dual_logreg(int n_samples, floating alpha, floating * theta,
     """Compute dual objective value at theta, which must be feasible."""
     cdef int i
     cdef floating d_obj = 0.
+    alpha = 1.
+
     for i in range(n_samples):
         d_obj -= Nh(alpha * y[i] * theta[i])
     return d_obj
 
-
+# remove alpha
+# [before] should work with alpha = 1
 cdef floating dual(int pb, int n_samples, floating alpha, floating norm_y2,
                    floating * theta, floating * y) nogil:
+
+    alpha = 1.
     if pb == LASSO:
         return dual_lasso(n_samples, alpha, norm_y2, &theta[0], &y[0])
     else:
         return dual_logreg(n_samples, alpha, &theta[0], &y[0])
 
 
+# remove alpha
```

Comment:
can you avoid comments like these which when you don't need them anymore, which make the diff longer to review ?